dallinger/frontend/static/scripts/dallinger.js

```diff
@@ -13,6 +13,31 @@ var getUrlParameter = function getUrlParameter(sParam) {
         }
     }
 };
+
+var Dallinger = (function () {
+  var dlgr = {};
+
+  participantId = getUrlParameter("participant_id") || -1;
+
+  dlgr.submitQuestionnaire = function (name, callback) {
+    formSerialized = $("form").serializeArray();
+    formDict = formSerialized.reduce(
+      function(m,o){ m[o.name] = o.value; return m; }, {}
+    );
```

Comment:
Maybe this is just a personal character flaw, but I always have to pause for a minute to remember how `reduce` works. Can we do this instead?
```
var formDict = {};
formSerialized.forEach(function (field) {
    formDict[field.name] = field.value;
});
```